pom.xml

```diff
@@ -254,6 +261,14 @@
                 <groupId>org.asciidoctor</groupId>
                 <artifactId>asciidoctor-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>9</source>
```

Comment:
We are targeting java 8, not 9 right?

Reply:
Also, I would think this is part of the parent pom?

Reply:
I vaguely recall adding this when prompted by intellij to facilate some <class>.of(...) call.  Will check

Reply:
removed. It was for Set.of(...) in a test.